contracts/test/LightClient.t.sol

```diff
@@ -1153,9 +1153,19 @@ contract LightClient_HotShotCommUpdatesTest is LightClientCommonTest {
             lc.getHotShotCommitment(newState.blockHeight - 1);
         assertEqBN254(blockComm, newState.blockCommRoot);
         assertEq(blockHeight, newState.blockHeight);
+
+        // Test for max hotShotBlockHeight stored on contract
+        // Get the highest HotShot blockheight recorded
+        uint256 numCommitments = lc.getStateHistoryCount();
+        (,, blockHeight,) = lc.stateHistoryCommitments(numCommitments - 1);
+        lc.getHotShotCommitment(blockHeight);
```

Comment:
should we have an asset statement here? seems like we are only calling the function and not checking for anything

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/52629295adfdc947c8648cb8c96ac2f7643d7f1b